src/hooks/coreSDK/useInitSafeCoreSDK.ts

```diff
@@ -26,9 +26,10 @@ export const useInitSafeCoreSDK = () => {
       .catch((e) => {
         dispatch(
           showNotification({
-            message: `The Safe SDK could not be initialized. Please try connecting the wallet again.`,
+            message: `Please try connecting your wallet again.`,
```

Comment:
Thanks for the fix, @SandeepGumaste!

I would mention RPC in the message because that's what usually makes the SDK fail, e.g. 

```suggestion
            message: 'Problem connecting to the RPC provider. Please try connecting your wallet again.',
```